tests/topotests/multicast-pim-static-rp-topo1/test_multicast_pim_static_rp.py

```diff
@@ -112,11 +112,14 @@
 
 # Required to instantiate the topology builder class.
 
-# pylint: disable=C0413
+# pylint: disable=C0413,C0302,R0915,C0103
```

Comment:
:thinking:

Reply:
before I figured out about "-E", there were ... some style-ish warnings that I didn't think we were going to fix. do you want me to change that back?

Reply:
Yeah, let's leave them enabled so if someone wants to do a cleanup pass they'll see them

Reply:
ok, done